### ISSUE TITLE
Can't get GraphQl queries working

### ISSUE THREAD

**Issue by soulchainer** [Huachao/vscode-restclient#501]
I tried several times, with several options and I can't get GraphQl queries working.
REST works just fine.

I created a `gql.http` file similar to this:

```
# GraphQl queries

@baseUrl = my.url.com:3000/path/to/graphql
@auth = Bearer hereALongToken

POST http://{{baseUrl}} HTTP/1.1
X-Request-Type: GraphQL
Content-Type: application/x-www-form-urlencoded
Authorization: {{auth}}


query ($some_parameter_value: String!) {
  some_query_name (some_query_argument: { some_parameter: $some_parameter_value }) {
     some
     query
     fields
  }
}

{
  "some_parameter_value": "value for the parameter"
}
```

I always get something like this:

```
HTTP/1.1 400 Bad Request
Server: serverName/x.x.x (Linux)
Date: Thu, 30 Jan 2020 08:00:27 GMT
Content-Type: application/json;charset=utf-8
Transfer-Encoding: chunked
Connection: close

{
  "errors": [
    {
      "message": "GraphQL Request must include at least one of those two parameters: \"query\" or \"queryId\"",
      "category": "request"
    }
  ]
}
```

Am I doing something obvious wrongly and I'm not seeing it or what?
I tried passing directly the values in the query also, same result.
I'm using an alias in `/etc/hosts` for `127.0.0.1` (I create an alias with a domain), but even if I use the addresss directly, `127.0.0.1`, I get the same error.
I read about this extension configuration and I'm also clueless.

If you can bring some advice or help, it will be highly appreciated. Thanks for your time! And for the extension :D.

**Comment by Huachao** [comment]
@soulchainer how about setting the `Content-Type` header in request to `application/json`

**Comment by soulchainer** [comment]
First of all, thanks for your time, @Huachao :).

Well, now I can't even get to that point (with the previous `Content-Type` or the `application/json`), because I hit this error:

```
write EPROTO 64295967845896:error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER:../../third_party/boringssl/src/ssl/tls_record.cc:242:
```

![Captura de pantalla de 2020-02-03 10-54-38](https://user-images.githubusercontent.com/169524/73643403-b9c37800-4673-11ea-8243-50a3ee5f8857.png)

**Comment by Huachao** [comment]
@soulchainer did you mean if you revert back to `Content-Type: application/x-www-form-urlencoded` the error is still the same as following.

> {
  "errors": [
    {
      "message": "GraphQL Request must include at least one of those two parameters: \"query\" or \"queryId\"",
      "category": "request"
    }
  ]
}

**Comment by soulchainer** [comment]
No, @Huachao. I mean that it doesn't matter which of these `Content-Type` I chose, because I get now that `write EPROTO` error always. So it doesn't do anything more, it doesn't even get to the part where a response an `errors` response is returned.

**Comment by Huachao** [comment]
@soulchainer your request protocol is http not https, so the ssl error confused with me

**Comment by Huachao** [comment]
@soulchainer your request protocol is http not https, so the ssl error confused with me

**Comment by soulchainer** [comment]
@Huachao is the protocol where our local GraphQl is running on.
Anyways... I did nothing (well, I just updated today, like each morning, my system, `sudo apt-get update && sudo apt-get upgrade`) and it started to work out of the blue.

Thanks for your time again, @Huachao. And for this awesome extension :). I'm closing this now, as the problem is solved at the moment.

Special note, just in case: I did not changed anything in my request file, is the same that when I started this issue, just that now, it works, finally :þ. :tada:
